packages/perseus/src/util/graphie.ts

```diff
@@ -1005,13 +1041,6 @@ export class Graphie {
             };
             const clippedFn = (x) => clip(fn(x));
 
-            if (!this.currentStyle.strokeLinejoin) {
-                this.currentStyle.strokeLinejoin = "round";
-            }
-            if (!this.currentStyle.strokeLinecap) {
-                this.currentStyle.strokeLinecap = "round";
-            }
```

Comment:
Assuming this is unused code?

Reply:
Yeah, nothing is setting `strokeLinejoin` or `strokeLinecap`.